site/docs/hosting/comparison.md

```diff
@@ -0,0 +1,62 @@
+# Comparison of Hosting Providers
+
+There are many different hosting providers that allow you to run your bot.
+It can sometimes be hard to keep track of how much they cost and how good their performance is.
+This is why the grammY community is collecting their experiences on this page.
+
+## What is a Hosting Provider?
+
+In order to keep a bot online 24 hours a day, you need to run a computer 24 hours a day.
+As [mentioned in the introduction](/guide/introduction.html#how-to-keep-a-bot-running), you most likely don't want to do that with your laptop or home computer.
+Instead, you can ask a company to run the bot in the cloud.
+
+In other words, you just run it on someone else's computer.
+
+## Comparison Tables
+
+> Please click the edit button at the bottom of the page to add more providers or to edit existing ones!
+
+We have two comparison tables, one for [serverless](#what-does-serverless-mean) hosting and one for [VPS](#vps).
+
+### Serverless
+
+Serverless means that you do not control a single machine on which your bot is run.
+Instead, these hosting providers will rather allow you to upload your code, and then start and stop different machines as necessary to make sure that your bot always works.
+
+This has the downside that your bot does not have access to a persistent storage by default, such as a local file system.
+Instead, you will often have to have a database separately and connect to it if you need to store data permanently.
+We therefore recommend you to use a different kind of hosting for more complex bots, e.g. a [VPS](./vps.md).
+
+The main thing to know about them is that you must run your bot on webhooks on serverless infrastructure.
```

Comment:
Are we sure about this? Heroku can be used with polling bots, too.

Reply:
Heroku offers workers, they are different and do not count as serverless solutions. The are comparable to GAE instances.